data/data.json

```
@@ -2535,8 +2676,8 @@
       "alpha_3": "prt"
     },
     "tld": ".pt",
-    "constitutional_form": "republic"
-  },
+    "constitutional_form": "republic",
+    "language_codes": ["pt-PT"]},
```

```suggestion
    "language_codes": ["pt-PT"]
  },
```
Can we fix the formatting? (same for lines: 2680, 2698, 2716, 2734, 2752, 2770, 2788, 2806, 2824, 2842, 2860, 2878, 2896, 2914, 2932, 2950, 2968, 2986, 3004, 3022, 3040, 3058, 3076, 3094, 3112, 3130, 3148, 3166, 3202, 3220, 3238, 3256, 3274, 3292, 3310, 3328, 3346, 3364, 3382, 3400, 3418, 3436, 3454, 3472, 3490, 3508, 3526, 3544, 3562, 3580, 3598, 3616, 3634, 3652, 3670)